modules/gltools/src/context/context.js

```diff
@@ -34,12 +31,12 @@ export function assertWebGL2Context(gl) {
   assert(isWebGL2(gl), ERR_WEBGL2);
 }
 
-const CONTEST_DEFAULTS = {
+const CONTEXT_DEFAULTS = {
   // COMMON CONTEXT PARAMETERS
   // Attempt to allocate WebGL2 context
   webgl2: true, // Attempt to create a WebGL2 context (false to force webgl1)
   webgl1: true, // Attempt to create a WebGL1 context (false to fail if webgl2 not available)
-  throwOnFailure: true,
+  throwOnError: false,
```

Comment:
Is there documentation that needs to be updated?

Reply:
Never mentioned anywhere, never used anywhere. Just a ghost prop leftover from a prior refactor I'm guessing.